lib/ex_webrtc/dtls_transport.ex

```diff
@@ -384,11 +387,13 @@ defmodule ExWebRTC.DTLSTransport do
     {:ok, state}
   end
 
-  defp handle_ice_data({:data, _data}, state) do
-    Logger.warning(
-      "Received RTP/RTCP packets, but DTLS handshake hasn't been finished yet. Ignoring."
-    )
+  defp handle_ice_data({:data, data}, state) do
+    Logger.debug("""
+    Received RTP/RTCP packets, but DTLS handshake hasn't been finished yet. \
+    They will be processed after the completion of the handshake.\
+    """)
 
+    state = %{state | buffered_remote_rtp_packets: state.buffered_remote_rtp_packets ++ [data]}
```

Comment:
You could also append the packets `[data | state.buffered_remote_rtp_packets]` and then do `Enum.reverse` in `flush_buffered_remote_rtp_packets`. Probably negligible in terms of performance, but more idiomatic imho.